lib/src/page_list_viewport_gestures.dart

```diff
@@ -308,15 +341,28 @@ class DeprecatedPanAndScaleVelocityTracker {
 
   int _previousPointerCount = 0;
   int? _previousGestureEndTimeInMillis;
-  int? _previousGesturePointerCount;
 
   int? _currentGestureStartTimeInMillis;
   PanAndScaleGestureAction? _currentGestureStartAction;
   bool _isPossibleGestureContinuation = false;
 
+  // Variables needed to keep track of repeated input scroll acceleration
+  // Consider the swipe for for repeated input acceleration
+  bool _isPossibleAccelSwipe = false;
+  bool _previosLaunchedWithMomentum = true;
+  int _numberOfRepeatedAcceleratedSwipes = 0;
+  final int _maxTimeIntervalBtwRepeatedSwipes = 1000;
+  Offset _prevLaunchVelocity = Offset.zero;
+
   Offset get velocity => _launchVelocity;
+  // Initial scalar multiplying velocity in the momentum simulation
+  double get momentumSimInitialVelocitySclar => _momentumSimInitialVelocityScalar;
+  double get dragIncreaseScalar => _dragIncreaseScalar;
   Offset _launchVelocity = Offset.zero;
 
+  double _momentumSimInitialVelocityScalar = 1.0;
```

Comment:
Is there a reason that you've separated these private variables from their corresponding getters? If not, please keep related properties close together. Otherwise we're inviting unnecessary confusion and read-jumping.

Reply:
How close should they be? They are 2 lines apart.